spec/models/app_identifier_claim_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+RSpec.describe AppIdentifierClaim do
```

Comment:
added minimal tests for this model:
* showing the validations carry through from the underlying models
* showing that save depends on the underlying models
* showing that the post-save accessors are derived from the other models